### PR TITLE
add registry field to maintainers

### PR DESCRIPTION
**Pull request by joelwass**

### CODE REVIEW

File: db/package.js [hunk 1]
```diff
@@ -105,11 +110,14 @@ class PackageDbController {
...
     return this.db.collection('packages').updateOne({
       _id: ObjectId(packageId)
     }, {
-      $set: { maintainers }
+      $set: {
+        ...(maintainers && { maintainers }),
+        ...(owner && { owner })
+      }
     })
   }
```
Comment (stripedpajamas) [inline db/package.js]: I thought we decided to not have an explicit "owner" ? There are people who are authorized via registry and people authorized via invite. What is the purpose of the owner field?

Comment (joelwass) [reply db/package.js]: owner determines who can edit the percentages, or were we going off whoever is "source registry" ? doesn't matter to me, i think both will have challenges / benefits

Comment (joelwass) [reply db/package.js]: i'll enumerate them:

---- having an owner field:
* one person in charge of percentages 
* when project adds more people to registry, they probably aren't thinking about flossbank rights so would protect against surprises there
* probably mirrors most payout systems, we could ask open collective what they do but i assume there is an admin of an OC project who controls payouts.
* downside of having potentially out of date owner who has stopped maintaining the project

--- not having an owner field
* provides easy maintenance for whenever an old maintainer leaves the project, anyone who has source registry can update the revenue share 
* unless we email everyone that rev shares have changed, how will teams know that someone changed flossbank rev shares?
* allows us to easily mirror registry information

Comment (joelwass) [reply db/package.js]: removed owners field - re-review :)

**Review by stripedpajamas — APPROVED** [review]

**Comment by stripedpajamas** [comment]
LGTM :shipit:

**Pull request merged**